chain/ethereum/src/chain.rs

```diff
@@ -51,7 +51,7 @@ lazy_static! {
     static ref MAX_BLOCK_RANGE_SIZE: BlockNumber = env_var("GRAPH_ETHEREUM_MAX_BLOCK_RANGE_SIZE", 2000);
 
     /// Ideal number of triggers in a range. The range size will adapt to try to meet this.
-    static ref TARGET_TRIGGERS_PER_BLOCK_RANGE: u64 = env_var("GRAPH_ETHEREUM_TARGET_TRIGGERS_PER_BLOCK_RANGE", 100);
+    static ref TARGET_TRIGGERS_PER_BLOCK_RANGE: u64 = env_var("GRAPH_ETHEREUM_TARGET_TRIGGERS_PER_BLOCK_RANGE", 1000);
```

Comment:
I'm not sure I understood why this changed and how it will behave with template handlers for filters.

Reply:
https://github.com/graphprotocol/graph-node/blob/465e7ad1a9e90cd529620318e37688a2eb784579/graph/src/blockchain/polling_block_stream.rs#L379

With the increase in triggers, the ranges went down to 1-10 and this more or less doubles the ranges